packages/react/src/components/Cta/Cta.js

```diff
@@ -38,18 +38,29 @@ const CTA = ({ style, cta }) =>
       {iconSelector(cta.type)}
     </LinkWithIcon>
   );
+/**
+ * TEMPORARY sets icon based on link type
+ *
+ * @param {string} type cta type ( external | jump | local)
+ * @returns {*} cta type component
+ */
+const TEMP_iconSelector = type =>
+  type === 'external' ? Launch20 : type === 'jump' ? ArrowDown20 : ArrowRight20;
```

Comment:
I think this is what was really causing the error for us. We need to try to phase this out and focus on one direction by being consistent. Either `<ArrowDown20 />`, or `ArrowDown20`. I'd advocate for changing `ButtonGroup` to allow for the first to align them all, but that's just me.